testsuite/tests/parsing/reloc.ml

```diff
@@ -1,6 +1,5 @@
 (* TEST
-   flags = "-I ${ocamlsrcdir}/parsing -I ${ocamlsrcdir}/toplevel"
-   include ocamlcommon
+   flags = "-I ${ocamlsrcdir}/parsing -I ${ocamlsrcdir}/compilerlibs -open Ocamlcommon"
```

Comment:
Because of the way `* expect` is currently implemented, it doesn't fully respect up the `include ocamlcommon` modifier. It could probably be fixed, but I did not want to increase the diff in this PR too much.